linera-storage/src/db_storage.rs

```diff
@@ -179,10 +189,20 @@ where
         ChainStateView::load(context).await
     }
 
+    async fn contains_hash(&self, hash: CryptoHash) -> Result<bool, ViewError> {
```

Comment:
I'd call it `contains_value`, because both values and certificates are addressed by hash.

Also, for https://github.com/linera-io/linera-protocol/pull/1384 we'll actually want to know whether we have a particular _certificate_, so I think we'll need both functions, `contains_value` and `contains_certificate`.

Reply:
Ok, added both. Let me know how things stand.